docs/sources/enterprise/usage-insights/improved-search.md

```diff
@@ -0,0 +1,25 @@
++++
+title = "Sort dashboards using insights data"
+description = "Sort dashboards using insights data"
+keywords = ["grafana", "search", "sort" "enterprise"]
+aliases = ["/docs/grafana/latest/enterprise/usage-insights/improved-search.md"]
+weight = 400
++++
+
+# Sort dashboards using insights data
+
+> **Note:** Available in Grafana Enterprise v7.0+.
+
+In the search view, you can sort dashboards using these insights data. It helps you find unused or broken dashboards or discover most viewed ones.
+
+The available sort options are:
+- Errors total
+- Errors 30 days
+- Views total
+- Views 30 days
+
+`Errors 30 days` and `Views 30 days` are based on a calculated sorting index that puts more weight into errors and views that happened on the past day than on the ones that happened on the past 30 days.  
```

Comment:
@mjseaman I don't know if we want to mention this as we're thinking about removing these sort options but good documentation around it could be a good start to know if users are interested on this.

If we want to keep it, I'd like some help to formulate this. We use this formula: `views_last_1_days * 14 + views_last_7_days + views_last_30_days / 6`, I don't know if we can explain it in plain text or if we want to put the formula in the documentation.